.github/workflows/website.yml

```diff
@@ -0,0 +1,27 @@
+name: Website
+
+on:
+  release:
+    types: [created]
```

Comment:
This should trigger the workflow only when a new release is created @oblador 
More info: https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#release